addons/mail/static/src/new/messaging.js

```diff
@@ -472,7 +472,7 @@ export class Messaging {
 
     async updateMessage(messageId, body) {
         const message = this.messages[messageId];
-        if (htmlToTextContentInline(message.body) === body) {
```

Comment:
Looks like this is the specific LOC that is for the fix.
The rest is just move of `convertBrToLineBreak` from `Composer` method to utils.